Sources/ParseSwift/Authentication/3rd Party/ParseInstagram/ParseInstagram+async.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ParseInstagram+async.swift
+//  ParseSwift
+//
+//  Created by Ulaş Sancak on 06/19/22.
+//  Copyright © 2022 Parse Community. All rights reserved.
+//
+
+#if compiler(>=5.5.2) && canImport(_Concurrency)
+import Foundation
+
+public extension ParseInstagram {
+    // MARK: Async/Await
+
+    /**
+     Login a `ParseUser` *asynchronously* using Instagram authentication.
+     - parameter id: The `Instagram profile id` from **Instagram**.
```

Comment:
When referring to "Instagram profile id" or similar, can you change to `**Instagram profile id**`. The former usually links to the types in the SDK. You can make this change on all of the added methods.

Look [here](https://parseplatform.org/Parse-Swift/release/documentation/parseswift/parsegoogle) to see how this shows up in the documentation.

Reply:
Will do. Thanks.